en/peripherals/dshot.md

```diff
@@ -19,13 +29,18 @@ DShot comes with different speed options: *DShot150*, *DShot300*, *DShot600* and
 You should set the parameter to the highest speed supported by your ESC (according to its datasheet) and then reboot the vehicle.
 
 Then connect the battery and arm the vehicle.
-The ESCs should initialize and the motors turn.
-If the motors do not spin at lowest throttle, increase [DSHOT_MIN](../advanced_config/parameter_reference.md#DSHOT_MIN) until they spin.
+The ESCs should initialize and the motors turn in the correct directions.
+- If the motors do not spin in the correct direction (for the [selected airframe](../airframes/airframe_reference.md)), reverse them by sending an [ESC Command](#commands).
+- Adjust [DSHOT_MIN](../advanced_config/parameter_reference.md#DSHOT_MIN) so that the motors spin at lowest throttle (but the vehicle does not take off).
+
+On boards that have both MAIN and AUX ports, set [SYS_USE_IO=0](../advanced_config/parameter_reference.md#SYS_USE_IO).
 
-> **Tip** On boards with an IO, DShot is only supported on the AUX pins.
-> It is best to disable the IO via [SYS_USE_IO](../advanced_config/parameter_reference.md#SYS_USE_IO) and then connect the motors to the AUX pins instead of MAIN.
+> **Note** DShot is only supported on FMU pins, which map to the AUX ports on boards that have both MAIN and AUX.
+  Setting `SYS_USE_IO=0` disables the IO board, and remaps the FMU pins to the MAIN port.
```

Comment:
> and remaps the FMU pins to the MAIN port

There is no physical remapping: the FMU pins still go to AUX (according to the board label). What is different is that the mixer for MAIN is loaded on the pins for AUX. That is why you have to replug your motors from MAIN to AUX.

Reply:
@bkueng Bugger. Thought I understood this from http://dev.px4.io/master/en/hardware/reference_design.html#mainio-function-breakdown

So what I previously understood was that .... if you had a board with only an FMU this only had MAIN ports and this would be labelled MAIN. If you had a board with FMU (AUX) and IO (MAIN) and you set `SYS_USE_IO=0` then the IO board is disabled and the ports labelled MAIN magically get remapped to whatever was on AUX. 

But what I think you are now saying is that (please confirm):
- FMU is always connected to AUX physical port. AUX is the "name" of the FMU port.
- IO is always connected to MAIN physical port. Main is the "name" of the IO port.
- On FMU-only system the FMU is connected to the AUX port (still) but it will load the MAIN mixer and will be *screen printed with MAIN*. The screen printing is what tells the user where they connect their wires.
- On FMU and AUX board with  `SYS_USE_IO=1`  we have FMU port AUX loaded with AUX mixer (screen printed AUX), IO port MAIN loaded with MAIN mixer (screen printed MAIN)
- On FMU and AUX board after setting `SYS_USE_IO=0`  we have FMU port AUX loaded with **MAIN** mixer (screen printed **AUX**), IO port MAIN disabled (screen printed MAIN)

So basically a user has to connect to the ports screen printed AUX as though they were labelled MAIN. Correct?

I wonder if we could fix this by naming port and mixer separately. Is there another generic way to refer to FMU AUX port that conveys this is a bank or PWM outputs pins? Ie FMU PWM Port?

Reply:
> On FMU-only system the FMU is connected to the AUX port (still) but it will load the MAIN mixer and will be screen printed with MAIN. The screen printing is what tells the user where they connect their wires.

You can call it that way if you want to. But FMU-only boards simply don't have any notion of AUX.

The rest is all correct.

> I wonder if we could fix this by naming port and mixer separately. Is there another generic way to refer to FMU AUX port that conveys this is a bank or PWM outputs pins? Ie FMU PWM Port?

Maybe just FMU Outputs/Port? If you add PWM in the name, it won't be correct for DShot.